apps/io-func-sign-user/CancelSignatureRequest/function.json

```
@@ -0,0 +1,15 @@
+{
+    "bindings": [
+      {
+        "authLevel": "function",
+        "type": "queueTrigger",
+        "direction": "in",
+        "name": "cancelSignatureRequest",
+        "queueName": "waiting-for-signature-request-cancelation",
```

```suggestion
        "queueName": "waiting-for-signature-request-cancellation",
```
The reference to the infrastructure (io-infra) must be added to this PR to create the new queue

---

do you need to create the queue using terraform before deploying this code?

---

PR here: https://github.com/pagopa/io-infra/blob/main/src/domains/sign/storage.tf